src/System.Net.Mail/tests/Functional/AttachmentTest.cs

```diff
@@ -32,6 +32,23 @@ public void ConstructorNullName()
             Assert.Equal(null, attach.Name);
         }
 
+        [Fact]
```

Comment:
Thanks for adding it - could you also add similar test for `Attachment(string fileName, string mediaType)` and `Attachment(string fileName, ContentType contentType)`?